### ISSUE TITLE
line comments don't get turned off from the configuration file setting.

### ISSUE THREAD

**Issue by chriseppstein** [Compass/compass#57]
From this mailing list thread: 
http://groups.google.com/group/compass-users/browse_thread/thread/9a030ab76c0cc436?hl=en

**Comment by chriseppstein** [comment]
fixed. turns out you couldn't effectively set any boolean property to false.

**Comment by felipero** [comment]
Weird thing is I'm getting this same problem today with config.rb file in rails.

**Comment by rszczypka** [comment]
I'm getting it too when trying to set 'line_comments = false'

**Comment by markwyner** [comment]
Howdy, folks.

I'm running a current version of Compass (0.12.2) and this still doesn't work unless I set line_comments to 0. False does nothing. Chris, 3 years ago you mention this was fixed. Is there any information about when this will make it into a release? Technically using boolean values are fine, but the documentation states false will work.

I'd love to see Compass either change its documentation or update the gem.

Thoughts?

Thanks in advance.
